src/domain/flow-system/domain/src/entities/flow/flow_status.rs

```diff
@@ -17,4 +19,14 @@ pub enum FlowStatus {
     Finished,
 }
 
+impl Display for FlowStatus {
```

Comment:
Consider using `#derive(strum::Display)` instead

Reply:
Yup, forgot about this stuff.